README.md

```diff
@@ -34,18 +28,53 @@ npm install react-image --save
 
 ## Dependencies
 
-`react-image` has no external dependencies, aside for the usual `react` and `react-dom`.
+`react-image` has no external dependencies, aside for a version of `react` and `react-dom` which support hooks.
 
 ## Documentation
 
+You can use the standalone component, documented below, or the `useImage` hook.
+
+### useImage():
+
+The `useImage` hook allows for incorperating `react-image`'s logic in any component. When using the hook, the component can be wrapped in `<Suspense>` to keep it from rendering until the image is ready. Specify the `fallback` prop to show a spinner or any other component to the user while the browser is loading. The hook the throw an error if it failes to find any images. You can wrap your componenet with an [Error Boundry](https://reactjs.org/docs/code-splitting.html#error-boundaries) to catch this scenario and do/show something.
+
+Example usage:
+
+```js
+import useImge from 'react-image'
+
+export default function MyComponent() {
+  const {src, isLoading, error} = useImage({
+    srcList: 'https://www.example.com/foo.jpg"',
+  })
+
+  return <img src={src} />
+}
```

Comment:
example here is wrong or misleading.

`useImage` isnt actually exported. see suggestion on export file.

figure out export structure then update examples
i suggest

```
import Img, {useImage} from 'react-image'
```